### DISCUSSION TITLE
Why runonReady command does not work as well as it runs the same command in windows powershell?

### DISCUSSION THREAD

**Discussion by FogWhileGH** [bluenviron/mediamtx#1537] [Questions]
First command to push rtsp to rtsp-simple=server, and second command to read the rtsp stream on the server,than re-encodeing the stream  and finally push it to server.

ffmpeg -re -stream_loop -1 -i test3.mp4 -c copy -rtsp_transport tcp -f rtsp rtsp://localhost:8554/o_stream

ffmpeg -rtsp_transport tcp -i rtsp://localhost:8554/o_stream -max_muxing_queue_size 1024 -vcodec h264_nvenc -bf 0 -acodec libopus -strict -2 -f rtsp rtsp://localhost:8554/mystream

If the second command is wrote as runonReady command, the server will kill the cmd, but it will run continually and finally push a high-qulaty stream to the server if it is running in the windows powershell.

version v0.21.5_windows_amd64

**Comment by aler9** [comment]
Hello, commands written inside `runOnReady` are executed directly without passing through PowerShell or Cmd.
If you think that a command behaves different when it is run inside PowerShell, use `runOnReady` to launch PowerShell and the command inside it:

```yml
paths:
  mypath:
    runOnReady: powershell -command 'ffmpeg -re -i ....'
```

[a comment by an automated account is not shown]
